Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt

```diff
@@ -104,36 +105,34 @@ fun Home() {
 
 @Composable
 fun HomeAppBar(
+    backgroundColor: Color,
     modifier: Modifier = Modifier
 ) {
-    Stack(modifier = modifier) {
-        ProvideEmphasis(EmphasisAmbient.current.high) {
-            Icon(
-                asset = vectorResource(R.drawable.ic_text_logo),
-                modifier = Modifier.align(Alignment.Center)
-                    .padding(8.dp)
-                    .preferredHeightIn(max = 24.dp)
-            )
-        }
-
-        ProvideEmphasis(EmphasisAmbient.current.medium) {
-            IconButton(
-                onClick = { /* TODO: Open account? */ },
-                modifier = Modifier.align(Alignment.CenterStart)
-                    .padding(start = 8.dp)
-            ) {
-                Icon(Icons.Default.AccountCircle)
+    TopAppBar(
+        title = {
+            Row {
+                Image(asset = vectorResource(R.drawable.ic_logo))
+                Icon(
```

Comment:
Why use `Icon` here? Feels like `Image` would be more fitting as you're not using the default sizing?

Reply:
It gives the automatic `contentColor`. Not a huge win, but good enough imo.